install.sh

```diff
@@ -102,14 +102,15 @@ function install() {
   NAMESPACE=${NAMESPACE:-$NAMESPACE_DEFAULT}
   LIQO_SUFFIX=${LIQO_SUFFIX:-$LIQO_SUFFIX_DEFAULT}
   LIQO_VERSION=${LIQO_VERSION:-$LIQO_VERSION_DEFAULT}
+  DASHBOARD_INGRESS=${DASHBOARD_INGRESS}
 
 
 
   #Wait for the installation to complete
   kubectl create ns $NAMESPACE || true
   $HELM_PATH dependency update $TMPDIR/liqo/deployments/liqo_chart
   $HELM_PATH install liqo -n $NAMESPACE $TMPDIR/liqo/deployments/liqo_chart --set podCIDR=$POD_CIDR --set serviceCIDR=$SERVICE_CIDR \
-    --set gatewayIP=$GATEWAY_IP --set global.suffix="$LIQO_SUFFIX" --set global.version="$LIQO_VERSION"
+    --set gatewayIP=$GATEWAY_IP --set global.suffix="$LIQO_SUFFIX" --set global.version="$LIQO_VERSION" --set global.dashboard_ingress=$DASHBOARD_INGRESS
```

Comment:
You should probably add the quotes around the variable name `--set global.dashboard_ingress="${DASHBOARD_INGRESS}"` (at least, shellcheck will tell you to do so).